.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
 default_language_version:
-  python: python3.9
+  python: python3
```

Comment:
Undo this change

Reply:
I changed it back now. Accidentally worked with 3.11 and got pre-commit not running without this change. Works now.